src/components/attribute/scroll/cdvy-list-on-scroll-bottom.directive.js

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
could you make a specific PR for this component (or a separate commit) with a dedicated JIRA issue (can be a sub-issue) as this component will be used by factories but it can be used without factories